src/test/java/CateringSystemTest.java

```diff
@@ -48,10 +48,10 @@ public void tearDown(){
 
     @Test
     public void testCreateEvent(){
-        c.createEvent("Birthday", new Date(121, Calendar.OCTOBER, 29, 18, 30, 24), "CN Tower",
+        c.createEvent("Birthday", new GregorianCalendar(2021, Calendar.OCTOBER, 29), "CN Tower",
                 30, "dinner");
         ArrayList<Event> Events = new ArrayList<Event>();
-        Event test_event = new Event(0, "Birthday", new Date(121, Calendar.OCTOBER, 29, 18, 30, 24), "CN Tower",
+        Event test_event = new Event(0, "Birthday", new GregorianCalendar(121, Calendar.OCTOBER, 29), "CN Tower",
```

Comment:
year is 121 (should it be 2021?)

Reply:
yep, and I just replaced 121 with 2021. Thanks!